centreon/www/front_src/src/ApiTokens/TokenCreation/Form.tsx

```diff
@@ -72,6 +74,16 @@ const FormCreation = ({
     values
   });
 
+  const platformFeatures = useAtomValue(platformFeaturesAtom);
+  const { canManageApiTokens, isAdmin } = useAtomValue(userAtom);
+
+  const getUsersEndpoint = (): string =>
+    platformFeatures?.isCloudPlatform && !isAdmin
+      ? getEndpointConfiguredUser({
+          search: { regex: { fields: ['is_admin'], value: '0' } }
```

Comment:
why we can t do the same thing for the canManageApiTokens , if the user  can create just his own token , we will search for the current user

Reply:
it will avoid unnecessary API call to `configuration/users` endpoint. we do have all necessary information to create a `SelectEntry` from `userAtom` information